packages/big-design/src/managers/alerts/spec.tsx

```diff
@@ -221,26 +221,26 @@ describe('alertsManager functionality', () => {
 });
 
 describe('AlertsManager', () => {
-  const successAlert = {
+  const successAlert: AlertProps = {
```

Comment:
You can always extend the type to ensure `key` is not undefined...
```ts
type KeyedAlertProps = AlertProps & {
  key: string;
};
```

That way you don't have to write conditionals